### PR TITLE
Add new crown to flow elements

### LINKED ISSUE #818: Update flow elements to use new crown
**Acceptance Criteria:**
Should only have trash icon.

Covers: 
- Sequence Flows
- Message Flow

Related: #789

### PR DESCRIPTION
**Pull request by tuantruong625**
The purpose of the PR is to implement the new crown to the flow elements (Sequence & Message). Also removes the old implementation of the crown on flows.

Fixes #818

### CODE REVIEW

File: src/components/crownConfig.vue [hunk 1]
```diff
@@ -109,7 +111,7 @@ export default {
   computed: {
     style() {
       const { x, y, width: shapeWidth } = this.shape.findView(this.paper).getBBox();
-      const width = (this.node.diagram.bounds.width * this.paper.scale().sx);
+      const width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
```
Comment (lisa-fehr) [inline src/components/crownConfig.vue]: You removed some code I put in to get the trash icon closer to the sequence/message flows. Now it subtracts 40.
```suggestion
->      let width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
      const widthOffsetFromLabel = shapeWidth - width > 3 ? (shapeWidth / 2) - 15 : 0;
      if ([
        'processmaker-modeler-sequence-flow',
        'processmaker-modeler-message-flow',
      ].includes(this.node.type)) {
        width = shapeWidth;
      }
      
      const widthOffsetFromLabel = shapeWidth - width > 4 ? (shapeWidth / 2) - 15 : 0;
      return {
        top: `${y - 45}px`,
->        left: `${x + width + widthOffsetFromLabel}px`,
        cursor: 'pointer',
      };
```

**Comment by lisa-fehr** [comment]
Might want to wait until pools are merged in to see how this looks. Trash placement can be odd.
Screen Shot 2019-11-19 at 1 24 53 PM

vs
Screen Shot 2019-11-19 at 1 25 06 PM

File: src/components/crownConfig.vue [hunk 2]
```diff
@@ -109,12 +111,19 @@ export default {
   computed: {
     style() {
       const { x, y, width: shapeWidth } = this.shape.findView(this.paper).getBBox();
-      const width = (this.node.diagram.bounds.width * this.paper.scale().sx);
-      const widthOffsetFromLabel = shapeWidth - width > 4 ? (shapeWidth / 2) - 15 : 0;
+      let width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
+      const widthOffsetFromLabel = shapeWidth - width > 3 ? (shapeWidth / 2) - 15 : 0;
```
Comment (JCook21) [inline src/components/crownConfig.vue]: The numbers in the calculation are pretty 'magical'. Could this be cleared up by extracting them to constants and giving them names that represent what they're for?

Comment (chipit24) [reply src/components/crownConfig.vue]: @JCook21 I removed the need for these constants in my PR, here https://github.com/ProcessMaker/modeler/pull/829/files#diff-c1b56e458c9fac3f4960d9c517f46cdbR195.

I think it's best to wait for that to be merged into `develop`, then this branch can be brought up to speed and use that logic here as well.

Comment (JCook21) [reply src/components/crownConfig.vue]: @chipit24 good plan, thanks.

**Comment by chipit24** [comment]
@tuantruong625 https://github.com/ProcessMaker/modeler/pull/829 has been merged. Once conflicts are resolved, and assuming all tests pass, this should be good to merge as well!

Keep in mind, when solving the merge conflicts, that I removed the `style` computed property and it's now a data property that's being updated through the `repositionCrown ` method.

File: src/mixins/linkConfig.js [hunk 3]
```diff
@@ -44,18 +44,19 @@ export default {
...
         this.shape.attr({
           line: { stroke: 'black' },
         });
-
         this.shapeView.hideTools();
       }
     },
   },
   computed: {
+    shapeView() {
```
Comment (lisa-fehr) [inline src/mixins/linkConfig.js]: I did this too on a pr I just merged into develop for association flows. But I put it at the bottom of computed. Just check if this tries to put it in twice before merging.

Comment (lisa-fehr) [reply src/mixins/linkConfig.js]: Never mind. It was going to add a duplicate. I fixed it.

**Pull request merged**